src/screens/ReceiveModal.js

```diff
@@ -74,12 +63,19 @@ export default function ReceiveModal() {
   }, []);
 
   const checksummedAddress = useMemo(() => toChecksumAddress(accountAddress), [accountAddress]);
+  const { height: deviceHeight } = useDimensions();
+  const { top } = useSafeAreaInsets();
 
   return (
-    <Container backgroundColor="rgba(0,0,0,0.85)" testID="receive-modal">
-      <TouchableBackdrop onPress={goBack} />
-      <Handle />
-      <ColumnWithMargins align="center" margin={24}>
+    <SimpleSheet
+      testID="receive-modal"
+      backgroundColor={'rgba(0,0,0,0.85)'}
+      useAdditionalTopPadding={200}
+      customHeight={IS_ANDROID ? deviceHeight - top : deviceHeight - sharedCoolModalTopOffset}
+      scrollEnabled={false}
+      top={{ custom: safeAreaInsetValues.top + 6 }}
```

Comment:
some of these props could be cleaned up. I don't think top is an accepted prop into the SimpleSheet component